CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## unreleased
 * BraintreePayPal
   * Add PayPal vault edit flow (BETA)
-    * Add `BTPayPalClient.edit(_:completion:)` method that takes in a `BTPayPalVaultEditRequest` or a `BTPayPalVaultErrorHandlingEditRequest`
+    * Add `BTPayPalClient.edit(_:)` method that takes in a `BTPayPalVaultEditRequest` or a `BTPayPalVaultErrorHandlingEditRequest`
```

Comment:
Looks like this is how apple documents async/await methods. Used this as a reference: https://developer.apple.com/documentation/foundation/urlsession/3767353-data